src/samples/article-vor/v6/complete.json

```diff
@@ -1107,8 +1107,7 @@
                 }
             ],
             "articleTitle": "Minimal journal article",
-            "journal": "A journal",
-            "pages": "In press"
```

Comment:
Was the `pages` property removed in error?

Reply:
There's some weird display issue here because I did both a `git rm src/samples/article-vor/v4/complete.json` and I added `/v6/complete.json` which is an exact copy of `/v5/complete.json`, this is wrongly concluding the v4 has been turned into v6. Therefore, if this data was removed, it was at the time the `/v5/complete.json` file was added.

Reply:
That makes sense. I consider this resolved.